examples/framework.rs

```diff
@@ -33,9 +33,14 @@ pub fn load_glsl(code: &str, stage: ShaderStage) -> Vec<u8> {
     };
 
     let mut output = glsl_to_spirv::compile(&code, ty).unwrap();
-    let mut spv = Vec::new();
-    output.read_to_end(&mut spv).unwrap();
-    spv
+    let mut spv_bytes = Vec::new();
+    output.read_to_end(&mut spv_bytes).unwrap();
+
+    let mut spv_words = Vec::new();
+    for bytes4 in spv_bytes.chunks(4) {
```

Comment:
fyi, we could re-export [read_spirv](https://docs.rs/gfx-hal/0.3.0/gfx_hal/pso/fn.read_spirv.html) from gfx-hal and use here